src/amo/components/Header/styles.scss

```diff
@@ -7,7 +7,7 @@
 .Header {
   background: $ink-90;
   display: grid;
-  grid-template-columns: min-content auto;
+  grid-template-columns: auto auto auto min-content;
```

Comment:
Maybe we could use `repeat(3, auto) min-content`

Reply:
Ah, cool, I'll try it. At some point we should redo this grid to use named areas. It's pretty ridiculous with numbered columns but I didn't want to tackle it in this PR.